client/signup/config/flows-pure.js

```diff
@@ -135,6 +108,14 @@ export function generateFlows( {
 			showRecaptcha: true,
 		},
 
+		'quick-onboard': {
+			steps: [ 'user', 'domains', 'plans' ],
+			destination: getSignupDestination,
+			description: 'The improved onboarding flow.',
```

Comment:
Maybe the description can link to the master thread for more details?

Reply:
Fixed.